File: CSharp/ShopTest/Items/ItemBatchTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Xunit;

namespace ShopTest.Items {
    using Shop.Items;
    public class ItemBatchTest {
        [Fact]
        public void ItemAndStock_EqualsGiven() {
            Item i = new Item("Blue Shoes", 50);
            ItemBatch batch = new ItemBatch(i, 100);

            Assert.Equal(i, batch.Item);
            Assert.Equal(100, batch.Stock);
        }
    }
}

File: Java/TddShop/TddShopTest/src/Item/ItemBatchTest.java
package Item;

import org.junit.Assert;
import org.junit.Test;
import static org.junit.Assert.assertEquals;
public class ItemBatchTest {
    @Test
    public void ItemAndStock_EqualGiven(){
        Item i = new Item("Blue Shoes", 35);
        ItemBatch batch = new ItemBatch(i, 100);

        assertEquals(i, batch.Item());
        assertEquals(100, batch.Stock);
    }
}
